roles/vdm/tasks/sas_deployment_cr.yaml

```diff
@@ -1,161 +1,27 @@
+# These tasks are responsible for generating the SASDeployment Custom Resource
+
 ---
-- name: orchestration - Set orchestration tooling facts
+- name: sas deployment cr - Set orchestration tooling facts
```

Comment:
There are several references in the task names to `sas deployment cr` is the the SAS Deployment Container Registry? I know it's not, but the `cr` abbreviation is currently used to stand for just that. If this has another meaning let's find a better acronym. Too many below with the same issue. I'll just leave this one comment.

Reply:
I can see that task name being confusing and it does conflict with the  word "container registry". I think I'll forgo using an abbreviation for the task names and file name and change it to the whole name.

new file name: `roles/vdm/tasks/sasdeployment_custom_resource.yaml`
example task name update: `- name: sasdeployment custom resource- Set orchestration tooling facts`

Lines up better with the SAS documentation here: https://go.documentation.sas.com/doc/en/itopscdc/v_037/dplyml0phy0dkr/p0nid9gu3x2cvln1pzpcxa68tpom.htm

Reply:
Updates made, ready for re-review.